src/main/java/qna/domain/Answer.java

```diff
@@ -49,11 +49,12 @@ public Answer(Long id, User writer, Question question, String contents) {
 
         this.writer = writer;
         this.question = question;
-        this.contents = contents;
+        this.contents = new Contents(contents);
+        this.deleted = new Deleted(false);
     }
 
     public boolean isOwner(User writer) {
-        return this.writer.equals(writer);
+        return this.writer.getUserId().equals(writer.getUserId());
```

Comment:
```suggestion
        return this.writer.isSameUser(writer);
```

이러한 방식으로 객체에게 물음을 던져보는식으로 생각&연습해보시면 좋을 것 같습니다.

전체적으로 특정 기능이나 역할을 담당할 수 있는 존중받는 객체로 생각하고 책임을 부여하는게 좋을 것 같습니다.